tests/src/test/java/alluxio/client/fs/FileSystemMasterIntegrationTest.java

```diff
@@ -658,7 +658,8 @@ public void ttlExpiredCreateFile() throws Exception {
     mFsMaster.createDirectory(new AlluxioURI("/testFolder"), CreateDirectoryContext.defaults());
     long ttl = 1;
     CreateFileContext context = CreateFileContext.mergeFrom(CreateFilePOptions.newBuilder()
-        .setCommonOptions(FileSystemMasterCommonPOptions.newBuilder().setTtl(ttl)));
+        .setCommonOptions(FileSystemMasterCommonPOptions.newBuilder().setTtl(ttl)))
```

Comment:
can you elaborate on why this test is still behaving like DELETE?

Reply:
Do you how we can directly check whether we have the file in UFS?